vercel-functions/prerender/_prerender-ember-route.func/index.js

```diff
@@ -0,0 +1,63 @@
+import fs from 'fs';
+import FastBoot from 'fastboot';
+
+function parsePrerenderPath(request) {
+  const {
+    url,
+    query: { suffix },
+  } = request;
+
+  let prefix = (url || '/')
+    .replace(/\?(.*)$/, '') // strip query parameters
+    .replace(/^\/prerender\//, '/') // strip /prerender
+    .replace(/([^^])\/$/, '$1'); // strip trailing /
+
+  if (suffix) {
+    prefix = prefix.replace(/((\/[^/]+))$/, '') || '/'; // strip wildcard function name
```

Comment:
@VasylMarchuk I'm kinda confused with this - can you explain a bit more what's going on here? What's a "wildcard function name"?

Reply:
One tip if it helps: I've found it useful to include a brief description of the inputs / outputs of a function that does parsing/transformation. Something like: 

```
// 'https://google.com/path?query=value' -> '/path'
function extractPath() {}
```

Reply:
Will add a JSDoc comment!

Reply:
Moved the function into a separate module and added JSDoc